### PR TITLE
Updates ScriptTuner README

### PR DESCRIPTION
**Pull request by KipHamiltons**
Also fixes a typo

### CODE REVIEW

File: module/behaviour/tools/ScriptTuner/README.md [hunk 1]
```diff
@@ -13,28 +13,35 @@ started and the file will be created when it is saved.
...
+* , (comma): Go to previous frame
+* . (period): Go to next frame
+* Enter: Edit selected field
+* Space: Toggle motor lock
+* : (colon): List available commands
+* A: Save script as
+* G: Allows multiple gain edits at once
+* I: Delete current frame
+* J: "jump" to frame without the robot moving
+* M: Mirror script - flip about Z axis
```
Comment (ysims) [inline module/behaviour/tools/ScriptTuner/README.md:27]: Sagittal plane is more accurate than z axis

File: module/behaviour/tools/ScriptTuner/README.md [hunk 2]
```diff
@@ -13,28 +13,35 @@ started and the file will be created when it is saved.
...
-* N: insert a new frame before the current frame
-* D: delete current frame
+* X: Shutdown powerplant/stop binary
 
 ## Consumes
 
-* `CommandLineArgumentis` containing the name of the script to edit
-* `message::DarwinServos` containing the current positions of each servo
+* `NUClear::message::CommandLineArguments` containing the name of the script to edit
+* `message::platform::RawSensors` to get position of specific servo when locking it
```
Comment (ysims) [inline module/behaviour/tools/ScriptTuner/README.md:38]: Consumes a LockServo, but maybe mention this only happens within ScriptTuner.

File: module/behaviour/tools/ScriptTuner/README.md [hunk 3]
```diff
@@ -13,28 +13,35 @@ started and the file will be created when it is saved.
...
 
 ## Emits
 
-* `message::ServoWaypoint>` to control servos
+* `message::motion::ServoTarget` to control servos
+* `message::motion::ServoTargets` to control servos
+* `extension::ExecuteScript` to play the script
```
Comment (ysims) [inline module/behaviour/tools/ScriptTuner/README.md:44]: RegisterAction

File: module/behaviour/tools/ScriptTuner/README.md [hunk 4]
```diff
@@ -13,28 +13,35 @@ started and the file will be created when it is saved.
...
 
-* `CommandLineArgumentis` containing the name of the script to edit
-* `message::DarwinServos` containing the current positions of each servo
+* `NUClear::message::CommandLineArguments` containing the name of the script to edit
+* `message::platform::RawSensors` to get position of specific servo when locking it
 
 ## Emits
 
-* `message::ServoWaypoint>` to control servos
+* `message::motion::ServoTarget` to control servos
```
Comment (ysims) [inline module/behaviour/tools/ScriptTuner/README.md:41]: This is done specifically on the currently selected servo when we lock it or unlock it.

File: module/behaviour/tools/ScriptTuner/README.md [hunk 5]
```diff
@@ -13,28 +13,35 @@ started and the file will be created when it is saved.
...
+* `message::motion::ServoTargets` to control servos
```
Comment (ysims) [inline module/behaviour/tools/ScriptTuner/README.md:42]: This is done specifically on all servos when we move frames.

**Review by ysims — APPROVED** [review]
LGTM

File: module/behaviour/tools/ScriptTuner/README.md [hunk 6]
```diff
@@ -13,28 +13,38 @@ started and the file will be created when it is saved.
...
 The keyboard controls are as follows:
 
-* Up/Down arrow: select servo
-* Left/right arrow: select angle or gain
-* , (comma): go to previous frame
-* . (period): go to next frame
-* Enter: edit selected field
-* Space: toggle motor lock
-* S: save script
+* Up/Down arrow: Select servo
```
Comment (JosephusPaye) [inline module/behaviour/tools/ScriptTuner/README.md:16]: I prefer hyphens for lists in Markdown 😛 

```suggestion
- Up/Down arrow: Select servo
```

**Review by JosephusPaye — APPROVED** [review]
LGTM

**Pull request merged**